begin-transaction.md

```diff
@@ -24,16 +23,66 @@ No [privileges](privileges.html) are required to initiate a transaction. However
 In CockroachDB, the following are aliases for the `BEGIN` statement:
 
 - `BEGIN TRANSACTION` 
-- `START TRANSACTION` 
+- `START TRANSACTION`
+
+The following aliases also exist for isolation levels:
+
+- `REPEATABLE READ` is an alias for `SERIALIZABLE`
+- `READ UNCOMMITTED` and `READ COMMITTED` are aliases for `SNAPSHOT`
+
+For more information on isolation level aliases, see [Comparison to ANSI SQL Isolation Levels](transactions.html#comparison-to-ansi-sql-isolation-levelshttpsenwikipediaorgwikiisolationdatabasesystemsisolationlevels).
 
 ## Parameters
 
 | Parameter | Description |
 |-----------|-------------|
-|  |  |
+| `ISOLATION LEVEL` | If you don't want the transaction to run as `SERIALIZABLE` (CockroachDB's default, which provides the highest level of isolation), you can specify `SNAPSHOT`, which can provide better performance in high-contention scenarios.<br/><br/>For more information, see [Transactions: Isolation Levels](transactions.html#isolation-levels).<br/><br/>**Default**: `SERIALIZABLE` |
+| `PRIORITY` | If you don't want the transaction to run with `NORMAL` priority, you can set it to `LOW` or `HIGH`.<br/><br/>When transactions contend for access to the same resources, the transaction with the highest priority executes first; the lower priority transaction is then retried inheriting the priority of the "winning" transaction. This means the longer a transaction contends for resources, the more likely it is to succeed.<br/><br/>For more information, see [Transactions: Priorities](transactions.html#transaction-priorities).<br/><br/>**Default**: `NORMAL` |
```

Comment:
I'd leave out details like the inheritance of priorities here and just say that high priority transactions are less likely to be restarted than low priority ones. (but it's not guaranteed - low priority transactions can sometimes win). Save the rest of the details for the priorities section.